jetty-server/src/main/java/org/eclipse/jetty/server/session/JDBCSessionDataStore.java

```diff
@@ -764,10 +727,9 @@ protected void doInsert (String id, SessionData data)
 
                 ByteArrayOutputStream baos = new ByteArrayOutputStream();
                 ObjectOutputStream oos = new ObjectOutputStream(baos);
-                oos.writeObject(data.getAllAttributes());
+                SessionData.serializeAttributes(data, oos);
```

Comment:
`ObjectOutputStream` not closed, use try-with-resources.

Reply:
OK